src/test/java/it/eng/idsa/businesslogic/util/TestUtilMultipartMessageBuilder.java

```diff
@@ -0,0 +1,49 @@
+package it.eng.idsa.businesslogic.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import de.fraunhofer.iais.eis.Message;
+import it.eng.idsa.multipart.domain.MultipartMessage;
+
+public class TestUtilMultipartMessageBuilder {
```

Comment:
@davidjovanovic 
What is the purpose for this class?
MultipartMessageBuilder is already doing this

Reply:
Had an idea. Accidentally left it there, I will delete it.